src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -299,26 +294,16 @@ func InConfigDir(filename string) (string, string, error) {
 	return cdir, filepath.Join(cdir, filename), nil
 }
 
-func (cfg *Config) GetTrustedCACerts() *x509.CertPool {
-	pool := <-poolCh
-	if len(poolCh) == 0 {
-		poolCh <- pool
-	}
-	return pool
-}
-
-func (cfg *Config) createCertPool() error {
+func (cfg *Config) GetTrustedCACerts() (pool *x509.CertPool, err error) {
```

Comment:
`GetTrustedCACerts()` is called only during initialization or when new config available, no need to cache the certs. So move `createCertPool()` here and remove the channel which causes block.